toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -702,6 +708,14 @@ void FarmServer::queryHwInfo(HwInfo &hwInfo)
 	len = sizeof(physMemSize);
 	sysctl(mib, 2, &physMemSize, &len, NULL, 0);
 
+#elif defined(LINUX)
+	physMemSize = (TINT64)sysconf(_SC_PHYS_PAGES) * (TINT64)sysconf(_SC_PAGE_SIZE);
```

Comment:
I was telling you about virtual memory because you currently just set everything to 500 MB (lines 723-725) and that’s quite wrong.